<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbManagerDlgExportVector</class>
 <widget class="QDialog" name="DbManagerDlgExportVector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to vector file</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Output file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editOutputFile">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnChooseOutputFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="chkSourceSrid">
          <property name="text">
           <string>Source SRID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editSourceSrid">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkTargetSrid">
          <property name="text">
           <string>Target SRID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editTargetSrid">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkEncoding">
        <property name="text">
         <string>Encoding</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboEncoding">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkDropTable">
        <property name="text">
         <string>Drop existing one</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkSourceSrid</tabstop>
  <tabstop>editSourceSrid</tabstop>
  <tabstop>chkEncoding</tabstop>
  <tabstop>cboEncoding</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbManagerDlgExportVector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSourceSrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>editSourceSrid</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTargetSrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>editTargetSrid</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEncoding</sender>
   <signal>toggled(bool)</signal>
   <receiver>cboEncoding</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbManagerDlgExportVector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
